api/meetups/__init__.py

```diff
@@ -0,0 +1,6 @@
+meetup_data = {
```

Comment:
@Ogutu-Brian Why did you use pre-loaded data?

Reply:
I use this for test. Let me see if I can use a json file for the test data instead.

Reply:
Hello @Bakley ,
I have added the json file for storing test data for meetups

Reply:
Okay, great work